api/src/main/scala/com/gu/adapters/http/AvatarServlet.scala

```diff
@@ -80,6 +80,20 @@ class AvatarServlet(store: AvatarStore, publisher: Publisher, props: AvatarServl
     NotImplemented(ErrorResponse("Endpoint needs to be specified"))
   }
 
+  get("/service/data/:userId") {
+    NotImplemented(ErrorResponse("Endpoint needs to be specified"))
+  }
+
+  apiDelete("/service/data/:userId", operation(deleteUserPermanently)) { auth =>
+    val dryRun = params.get("dryRun").contains("true")
```

Comment:
I'll remove the endpoints since we'll be using SQS

Reply:
We can leave it then. Might be worth creating a queue shovel mechanism for retries, but not worth doing until we need it.